source/deploy/server/preparations.rst

```diff
@@ -0,0 +1,183 @@
+Prepare your Mattermost Server environment
+===========================================
+
+This guide outlines the key preparation steps required before installing the Mattermost Server, focusing on setting up the database and file storage systems.
+
+Database preparation
+--------------------
+
+Mattermost requires a PostgreSQL database (version 13 or higher). While MySQL was previously supported, PostgreSQL is now the recommended and preferred database.
+
+.. important::
+
+    PostgreSQL v13+ is required for Mattermost server installations. MySQL support is being deprecated starting with Mattermost v11.
+
+
+1. Create an PostgreSQL server instance:
+
+   .. tab:: AWS
+   
+      .. code-block:: sh
+
+         sudo apt update
+         sudo apt install postgresql
+    
+   .. tab:: Azure
+   
+      .. code-block:: sh
+
+         sudo apt update
+         sudo apt install postgresql
+    
+   .. tab:: Ubuntu/Debian
+   
+      .. code-block:: sh
+
+         sudo apt update
+         sudo apt install postgresql
+
+   .. tab:: Red Hat/CentOS
+   
+      .. code-block:: sh
+
+         sudo dnf install postgresql-server
+         sudo postgresql-setup --initdb
+
+2. Create the Mattermost database and user:
+
+   a. Access PostgreSQL by running:
+
+      .. code-block:: sh
+
+         sudo -u postgres psql
+
+   b. Create the database:
+
+      .. code-block:: sql
+
+         CREATE DATABASE mattermost;
+
+   c. Create the Mattermost user with a secure password:
+
+      .. code-block:: sql
+
+         CREATE USER mmuser WITH PASSWORD 'mmuser-password';
+
+   d. Grant database access to the user:
+
+      .. code-block:: sql
+
+         GRANT ALL PRIVILEGES ON DATABASE mattermost to mmuser;
+
+   e. If using PostgreSQL v15.x or later, additional grants are required:
+
+      .. code-block:: sql
+
+         ALTER DATABASE mattermost OWNER TO mmuser;
+         GRANT USAGE, CREATE ON SCHEMA PUBLIC TO mmuser;
+
+3. Configure PostgreSQL for remote connections (if database is on a separate server):
```

Comment:
I wonder if we should make it more clear that this only needs to be done if the DB is not on the same server.

Reply:
Are you looking for something more than just:

>(if database is on a separate server)

Reply:
Sound good :+1: